### DISCUSSION TITLE
How to use intraday .CSV file?

### DISCUSSION THREAD

**Discussion by henriquepba** [kernc/backtesting.py#756] [Q&A]
I'm starting to test backtesting.py

I copied a standard code, but with the "from backtesting.test import GOOG" and it ran normally, the data is daily.

When importing a CSV in the intraday (5 minutes) the code does not run.

When I see the "head" of GOOG I notice that it has "date", "Open", "High", "Low", "Close" and "Volume"

When separating the columns correctly, I notice that my CSV has "date", "time", "Open", "High", "Low", "Close" and "Volume".

Could this additional column be the problem?

I looked for other tutorials, but I only find data manipulation in the daily or weekly, hardly in the intraday.

Does anyone know where I can find how to manipulate the CSV with intraday data to run the backtest?

The CSV file I'm trying to run is this:
https://drive.google.com/file/d/1UI6fHCu2h2Z-m2wDh2DnKhN_Wv55ahZj/view?usp=sharing

**Comment by kernc** [comment]
See https://www.chiark.greenend.org.uk/~sgtatham/bugs.html
